schemas/fingerprint-server-api.yaml

```diff
@@ -1414,6 +1416,15 @@ components:
               $ref: '#/components/schemas/RemoteControlResult'
             error:
               $ref: '#/components/schemas/ProductError'
+        developerTools:
```

Comment:
Is this signal generally available or activated on request? Has it already been released? I don't see it enabled for `elvo` on https://demo.fingerprint.com/playground

Reply:
warden is released, mgmt migrations for all customers not yet

Reply:
Let's wait for those then :)

Reply:
It's seems they are out, resolving